src/Microsoft.DotNet.XHarness.CLI/Microsoft.DotNet.XHarness.CLI.csproj

```diff
@@ -93,13 +82,14 @@
     </ItemGroup>
   </Target>
 
-  <Target Name="IncludeMlaunch" BeforeTargets="Build" DependsOnTargets="DownloadMlaunch">
+  <Target Name="IncludeMlaunch" BeforeTargets="Build" DependsOnTargets="ExtractMlaunch">
```

Comment:
I don't see this new target, did you add it?

Reply:
It is inside of the mlaunch NuGet in the `build/Microsoft.DotNet.Mlaunch.props` file (I was showing it in the morning)

Reply:
ok, thanks

Reply:
@greenEkatherine I can add a comment there:
```suggestion
  <!-- The ExtractMlaunch target comes from the Mlaunch NuGet package and places files in the obj/ dir -->
  <Target Name="IncludeMlaunch" BeforeTargets="Build" DependsOnTargets="ExtractMlaunch">
```